sdk/src/instruction.rs

```diff
@@ -86,14 +86,14 @@ pub enum InstructionError {
     /// the runtime cannot determine which changes to pick or how to merge them if both are modified
     DuplicateAccountOutOfSync,
 
-    /// The return value from the program was invalid.  Valid errors are either a defined builtin
-    /// error value or a user-defined error in the lower 32 bits.
-    InvalidError,
-
     /// Allows on-chain programs to implement program-specific error types and see them returned
     /// by the Solana runtime. A program-specific error may be any type that is represented as
     /// or serialized to a u32 integer.
     CustomError(u32),
+
+    /// The return value from the program was invalid.  Valid errors are either a defined builtin
+    /// error value or a user-defined error in the lower 32 bits.
+    InvalidError,
```

Comment:
@mvines Regarding [this question](https://github.com/solana-labs/solana/issues/8511#issuecomment-592190274), yes these enum changes are automatically detected and failed as differing ABI digest error of the distant `BankStatusDelta`: https://github.com/solana-labs/solana/pull/8012/commits/fdf091c440c28f5901b5232c4dfb2ec51e03f64e (this commit is for the reorder of enum in this PR).

CC: @jackcmay